README.md

```diff
@@ -1 +1,63 @@
-# rest-api
+# Housing rest-api
+
+We are building a neighbourhood collaboration site and we want to make a system to keep track of people, houses, and addresses of those houses.
+
+    Each person has a name, age and number of people in their household
+    Each house has an address and an owner
+    Each address has a postcode and street address
+
+The REST API will need to enable users to:
+
+    Store people, houses and addresses
+    Look up a house, it’s address and owner
+    Look up people in our neighbourhood within certain age brackets and with specific household sizes
+
+As the data is related (people own houses, houses have addresses) we opted for a relational database using SQL.
+Our database schema will look like this:
+
+    Table name owners   |    name    |    age    |    household_size    |   PRIMARY KEY owner_id    |    FOREIGN KEY house_id 
+                        |    Kai     |    22     |          5           |          101              |             1
+                        |    Paola   |    25     |          6           |          102              |             2
+
+    Table name addresses    |    PRIMARY KEY house_id   |    street_address    |    postcode  
+                            |             1             |   22 Koala Street    |    HF12 4DE       
+                            |             2             |    23 Bear Road      |    JG24 2FW      
+
+Data will be accessed using SQL queries. 
+
+API will be able to handle GET, POST, PATCH/PUT and DELETE requests.
+
```

Comment:
liked that you have the sql query here but maybe have that all on one section, and have all the api url stuff in its own section. That way the users don't have to look at the whole document and just the specific part they want.